helper-functions/taskProgress.js

```diff
@@ -0,0 +1,14 @@
+import { timeWas } from '../helper-functions/time-was';
+
+function percentageofDaysRemaining(startedOn, endsOn, completedDate) {
+  let startDate = timeWas(startedOn * 1000, true, endsOn * 1000);
+  let endDate = timeWas(endsOn * 1000, true, startDate * 1000);
```

Comment:
Since we are not initializing startDate,endDate and diffTime could we go for const here?